src/test/java/SYSC6/Project/UserControllerTest.java

```diff
@@ -57,59 +57,36 @@ void getUserByUsernameNotExists() throws Exception{
 
     @Test
     @Order(5)
-    void getUserByUsernameandPassword_WrongPass() throws  Exception{
-        this.mockMvc.perform(get("/rest/api/user/get/Nick/Wrong")).andExpect(status().isBadRequest());
-    }
-
-    @Test
-    @Order(6)
-    void getUserByUsernameandPassword_NotExist() throws  Exception{
-        this.mockMvc.perform(get("/rest/api/user/get/test/Wrong")).andExpect(status().isBadRequest());
-    }
-
-    @Test
-    @Order(7)
-    void getUserByUsernameandPassword_GoodCred() throws  Exception{
-        JSONObject obj = new JSONObject();
-        obj.put("username", "Test");
-        obj.put("password","Check");
-        this.mockMvc.perform(post("/rest/api/user/add").contentType(MediaType.APPLICATION_JSON).content(String.valueOf(obj)));
-        this.mockMvc.perform(get("/rest/api/user/get/Test/Check")).andExpect(status().isOk()).andExpect(content().json("{\"id\":3,\"username\":\"Test\",\"password\":\"Check\",\"role\":\"FREE_USER\"}"));
-    }
-
-
-    @Test
-    @Order(8)
     void createUser() throws Exception {
         JSONObject obj = new JSONObject();
-        obj.put("username","Nick");
+        obj.put("username","Moe");
```

Comment:
Lol

Reply:
had to change it to something, users are supposed to have different usernames so the get by username command was causing issues